.github/workflows/staging-sgx-restart.yml

```diff
@@ -0,0 +1,37 @@
+name: Staging SGX parachain
+
+on:
+  workflow_dispatch:
+    inputs:
+      action:
+        type: choice
+        description: Select the action
+        required: true
+        default: restart
+        options: 
+        - restart
+        - stop
+        - start
+
+jobs:
+  action:
+    runs-on: self-hosted #tee-staging (162.19.84.27)
+
+    steps:
+      - name: restarting the service
+        if: "${{ github.event.inputs.action == 'restart' }}"
+        run: |
+          pkill -9 integritee-service
```

Comment:
What about the parachain?

Reply:
I think the python script only responds to `SIGINT` and `SIGTERM`, so you need to send those signals, not `SIGKILL`

Reply:
Maybe we can connect the `SIGKILL` too?